.lintr

```diff
@@ -13,6 +13,5 @@ linters: linters_with_defaults(
     todo_comment_linter = NULL,
     undesirable_function_linter(c("mapply" = NA, "sapply" = NA, "setwd" = NA)),
     undesirable_operator_linter = NULL,
-    unnecessary_concatenation_linter(allow_single_expression = FALSE),
```

Comment:
Why are you removing this?

Reply:
I updated to R 4.3, installed latest lintr from GitHub, but now I get this error all the time. We can add it back once we know why this happens, but for the moment I could no longer stand seeing all these red flags ;-)

Reply:
Strange. It should work with the devel-_lintr_. We don't see any failures on lintr-main with R-devel.

At any rate, we just need to make sure that the local `.lintr` configs don't stray too far away from our GHA [workflow](https://github.com/easystats/workflows/blob/main/.github/workflows/lint.yaml) because we will see different lints locally versus on GitHub.

Reply:
wait, let me check if installation of lintr dev-version worked